package.json

```diff
@@ -176,6 +176,7 @@
     "lodash": "^4.17.21",
     "loglevel": "^1.4.1",
     "luxon": "^1.26.0",
+    "moment": "^2.29.1",
```

Comment:
Is moment.js an acceptable package to add?

Reply:
I think date-fns is preferred due to its ability to be tree-shaken

https://blog.bitsrc.io/date-fns-vs-momentjs-9833f7463751

Reply:
`luxon` is the current Date management package of choice. I would say use that over moment.